lib/tasks/auto_annotate_models.rake

```diff
@@ -3,7 +3,7 @@
 # NOTE: to have a dev-mode tool do its thing in production.
 if Rails.env.development?
   require "annotate"
-  task :set_annotation_options do
+  task set_annotation_options: :environment do
```

Comment:
Quoting @mattbrictson when I made this rubocop update on a different project:

> I think it was misguided for RuboCop to introduce the Rails/RakeEnvironment cop. I think we need to avoid it. It might make sense for apps that use Rake only for invoking Rails application code, but we are mostly using it for running linters, etc. where loading the application code does not provide any benefit and slows things down.

Reply:
☝️ That makes a lot of sense!